src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -194,14 +198,59 @@ public static Time parseTime(String time) throws ParseException {
         return new Time(trimmedTime);
     }
 
+    /**
+     * Parses a {@code String title} into a {@code Title}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code title} is invalid.
+     */
+    public static Title parseTitle(String title) throws ParseException {
+        requireNonNull(title);
+        String trimmedTitle = title.trim();
+        if (Verifier.isValidTitle(trimmedTitle)) {
```

Comment:
to be consistent with other classes, could consider making title, description and place separate classes with method isValid(), instead of using a Verifier